dev/prepare_db.pql

```diff
@@ -4,41 +4,73 @@ func run_sql(code) {
     force_eval( SQL( nulltype, code ))
 }
 
+func drop_table(t) {
+    run_sql("DROP TABLE IF EXISTS " + t)
+}
+
+func create_indices(tbl) {
+    tbl.add_index("id", true)
+    tbl.add_index("timestamp")
+    tbl.add_index(["id", "timestamp"])
+}
+
 // Cleanup
 func cleanup() {
-    run_sql("DROP TABLE IF EXISTS rating")
-    run_sql("DROP TABLE IF EXISTS tmp_rating")
-    run_sql("DROP TABLE IF EXISTS rating_del1")
-    run_sql("DROP TABLE IF EXISTS rating_update1")
-    run_sql("DROP TABLE IF EXISTS rating_update001p")
-    run_sql("DROP TABLE IF EXISTS rating_update1p")
-    run_sql("DROP TABLE IF EXISTS rating_del1p")
-    run_sql("DROP TABLE IF EXISTS rating_update50p")
+    drop_table("rating")
+    drop_table("tmp_rating")
+    drop_table("rating_del1")
+    drop_table("rating_update1")
+    drop_table("rating_update001p")
+    drop_table("rating_update1p")
+    drop_table("rating_del1p")
+    drop_table("rating_update50p")
```

Comment:
Should `.drop` on the table definition be a `preql` construct?

Reply:
There is already `remove_table_if_exists`, which does the same thing. But it doesn't print out the SQL.